generator/shared-autoconfig-properties/pom.xml

```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <!-- Does it make sense to add spring-cloud-gcp as parent? -->
```

Comment:
Although this is manual-written module, I feel it makes sense to have it in `generated` folder alongside with generated code:
- they can be in same released schedule, and `generator` will only contain internal process code.
- they can also share same parent pom and dependencies

What do you all think? @elefeint @diegomarquezp @emmileaf

Reply:
Having this module in the same place as the generated modules makes sense to me!

Reply:
Makes sense to me as well. I will move it to `generated`